### PR TITLE
Update README.md file (#1656)

### PR DESCRIPTION
**Pull request by nbougalis**
Reviews: @mDuo13, @yana-novikova, @vinniefalco

### CODE REVIEW

**Comment by vinniefalco** [comment]
Love it!!! :+1:

**Comment by vinniefalco** [comment]
At some point we should set up build badges in the README.md

File: README.md [hunk 1]
```diff
@@ -1,129 +1,112 @@
...
-* Ripple Primer - https://ripple.com/ripple_primer.pdf
-* Ripple Primer (Market Making) - https://ripple.com/ripple-mm.pdf
-* Ripple Gateway Primer - https://ripple.com/ripple-gateways.pdf
-* Consensus - https://wiki.ripple.com/Consensus
+To learn about how Ripple is transforming global payments visit
+[https://ripple.com/contact/](https://ripple.com/contact/)
 
 - - -
 
 Copyright © 2015, Ripple Labs. All rights reserved.
```
Comment (yana-novikova) [inline README.md]: should it be 2016?

File: README.md [hunk 2]
```diff
@@ -1,129 +1,112 @@
...
 
-Portions of this document, including but not limited to the Ripple logo, images
-and image templates are the property of Ripple Labs and cannot be copied or
-used without permission.
+Portions of this document, including but not limited to the Ripple logo,
+images and image templates are the property of Ripple Labs and cannot be
```
Comment (yana-novikova) [inline README.md]: Ripple, no Labs

Comment (nbougalis) [inline README.md]: Happy to make the changes you mention but it's my understanding that the company is, officially, Ripple Labs and "Ripple" is a DBA. Should we get an official opinion on this?

Comment (yana-novikova) [inline README.md]: Yeah, we are still Ripple Labs so can keep it as it is.

File: README.md [hunk 3]
```diff
@@ -1,129 +1,107 @@
...
-
-### Repository Contents
-
-#### ./bin
+# What is Ripple?
+
+The Ripple Consensus Ledger (RCL) is a secure distributed ledger that uses
+the [consensus process](https://www.youtube.com/watch?v=pj1QVb1vlC0) to
+settle transactions and is maintained by a network of computers over the
+internet, which collectively create Ripple network. Because of its distributed
```
Comment (mDuo13) [inline README.md]: > which collectively create Ripple network

I suggest striking this phrase entirely, then changing "it" to "the Ripple Consensus Ledger" in the following sentence.

File: README.md [hunk 4]
```diff
@@ -1,129 +1,107 @@
...
+nature, it does not require a central operator and offers transaction
+immutability and information redundancy. RCL holds the order book with bid/ask
+offers from payment initiators and market makers. Its path-finding algorithm
+enables it to find the lowest foreign exchange rate across all order books
```
Comment (mDuo13) [inline README.md]: > Its path-finding algorithm enables it to find the lowest foreign exchange rate across all order books and currency pairs.

One, this is a slight overpromise -- pathfinding is not guaranteed to find the lowest rate. Second, "enables it to find" is wordier than necessary. Third, this sounds like a good opportunity to advertise auto-bridging. Suggested rephrase:

> A path-finding algorithm finds competitive foreign exchange rates across all order books and currency pairs, even joining books to convert through the native cryptocurrency, XRP, to ensure tighter spreads.

File: README.md [hunk 5]
```diff
@@ -1,129 +1,107 @@
...
-#### ./bin
+# What is Ripple?
+
+The Ripple Consensus Ledger (RCL) is a secure distributed ledger that uses
+the [consensus process](https://www.youtube.com/watch?v=pj1QVb1vlC0) to
+settle transactions and is maintained by a network of computers over the
+internet, which collectively create Ripple network. Because of its distributed
+nature, it does not require a central operator and offers transaction
+immutability and information redundancy. RCL holds the order book with bid/ask
+offers from payment initiators and market makers. Its path-finding algorithm
```
Comment (mDuo13) [inline README.md]: > market makers

Probably best to use "liquidity providers" or "currency traders" instead.

File: README.md [hunk 6]
```diff
@@ -1,129 +1,107 @@
...
+and currency pairs.
+
+### Key Features
+#### Distributed
+- Direct account-to-account settlement with no central operator
+- Decentralized global market for competitive FX
+- Transaction immutability and information redundancy
+
+#### Secure
+- Transactions cryptographically signed using ECDSA and ED25519 algorithms
```
Comment (mDuo13) [inline README.md]: Technically the D should be lowercase in "Ed25519."

File: README.md [hunk 7]
```diff
@@ -1,129 +1,107 @@
...
+- Multi-signing capabilities
+
+#### Scalable
+- Capacity to process the world’s cross-border payments volume
+- Easy access to liquidity through a competitive FX marketplace
+
+### XRP Role on Ripple as a Bridge Currency
```
Comment (mDuo13) [inline README.md]: Suggest you shorten this header to:

> XRP as a Bridge Currency

File: README.md [hunk 8]
```diff
@@ -1,129 +1,107 @@
...
+Ripple enables banks to settle cross-border payments in real-time, with
+end-to-end transparency, and at lower costs. Banks can choose to either
```
Comment (mDuo13) [inline README.md]: > choose to either

Suggest you scratch this whole phrase, so the sentence reads, "Banks can provide liquidity for FX themselves or source it from third-parties.

File: README.md [hunk 9]
```diff
@@ -1,129 +1,107 @@
...
+provide liquidity for FX themselves or source it from third-party
+market makers.
```
Comment (mDuo13) [inline README.md]: Avoid "market makers" (e.g. as in the rewrite from my comment on the line above)

File: README.md [hunk 10]
```diff
@@ -1,129 +1,107 @@
...
+Liquidity providers need to maintain accounts with each counterparty for
+each currency – a capital- and time-intensive endeavor that spreads liquidity
+thin. Further, some transactions, such as exotic currency trades, will require
+multiple trading parties, who each layer costs to the transaction. Thin
+liquidity and many intermediary trading parties make competitive pricing
+challenging.
+
+![Flow - Direct](images/flow1.png)
+
+Market makers can bridge any currencies directly through XRP in a trade.
```
Comment (mDuo13) [inline README.md]: Rewrite this paragraph to use "traders" instead of "market makers."

File: README.md [hunk 11]
```diff
@@ -1,129 +1,107 @@
...
 Platform or IDE-specific project files.
 
-#### ./doc
+##### ./doc
 Documentation and example configuration files.
 
-#### ./src
+##### ./src
 Source code directory. Some of the directories contained here are
 external repositories inlined via git-subtree, see the corresponding
```
Comment (mDuo13) [inline README.md]: make "See the corresponding..." into a separate sentence.

File: README.md (hunk 1 above)
Comment (sublimator) [inline README.md]: We do still use labs last I asked (re: ripple-dot-net LICENSE)

Comment (sublimator) [inline README.md]: Oh, sorry, just saw comments below

File: README.md [hunk 12]
```diff
@@ -1,129 +1,93 @@
 ![Ripple](/images/ripple.png)
```
Comment (sublimator) [inline README.md]: badges ?

Comment (vinniefalco) [inline README.md]: We need them. The problem is they are all red...

Comment (nbougalis) [inline README.md]: There are some badges already.

**Pull request closed without merging**